app/views/application_searches/_application_search_filter.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= form_with url: search_url, model: application_search_filter, scope: :filter, method: :get, class: 'search', id: 'search_filter' do |f| %>
-  <p class="govuk-label govuk-label--s">Enter application information</p>
 
   <%= f.govuk_fieldset legend: { text: nil } do %>
     <div class="input-group">
       <%= f.govuk_text_field(
             :search_text,
-            label: { text: "For example, reference number or applicant's first or last name" },
+            label: { text: 'Enter any combination of client first name, last name, LAA reference', size: 's', class: 'govuk-!-margin-bottom-4' },
```

Comment:
Not necessarily now, but it'd be good to move this text to a locale file like all the other strings.